File: python-pr/52/python/code-quality-error-handling/file_writer_34.py
# {fact rule=code-quality-error-handling@v1.0 defects=0}
def write_file(filename, content):
    try:
        with open(filename, 'w') as f:
            f.write(content)
        return True
    except IOError as e:
        print(f"Write error: {e}")
        return False
# {/fact}

File: js-pr/56/javascript/code-quality-error-handling/file_writer_34.js
// {fact rule=code-quality-error-handling@v1.0 defects=0}
function writeFile(filename, content) {
    const fs = require('fs');
    try {
        fs.writeFileSync(filename, content);
        return true;
    } catch (error) {
        console.error(`Error writing file ${filename}:`, error.message);
        return false;
    }
}
// {/fact}